### PR TITLE
feat(dashboard): custom domains

### PR DESCRIPTION
**Pull request by onehassan**
resolves nhost/projects#26

### CODE REVIEW

File: dashboard/src/features/projects/custom-domains/settings/components/AuthDomain/AuthDomain.tsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+      // Note: The toast will handle the error.
+    }
+  }
+
+  return (
+    <FormProvider {...form}>
+      <Form onSubmit={handleSubmit}>
+        <SettingsContainer
+          title="Auth Domain"
+          description="It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout."
```
Comment (NesManrique) [inline dashboard/src/features/projects/custom-domains/settings/components/AuthDomain/AuthDomain.tsx:118]: This has to be removed or replaced by a correct description

File: dashboard/src/features/projects/custom-domains/settings/components/AuthDomain/AuthDomain.tsx [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+            placeholder="auth.mydomain.dev"
+            error={Boolean(formState.errors.auth_fqdn?.message)}
+            helperText={formState.errors.auth_fqdn?.message}
+            slotProps={{ inputRoot: { min: 1, max: 100 } }}
+          />
+          <div className="col-span-5 row-start-2">
+            <VerifyDomain
+              recordType="CNAME"
+              hostname={auth_fqdn}
+              value={`lb.${currentProject.region.domain}.nhost.run.`}
```
Comment (NesManrique) [inline dashboard/src/features/projects/custom-domains/settings/components/AuthDomain/AuthDomain.tsx:147]: This is not showing the correct link, I think it should be

```suggestion
              value={`lb.${currentProject.region.awsName}.${currentProject.region.domain}`}
```

File: dashboard/src/features/projects/custom-domains/settings/components/AuthDomain/AuthDomain.tsx [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+            placeholder="auth.mydomain.dev"
+            error={Boolean(formState.errors.auth_fqdn?.message)}
+            helperText={formState.errors.auth_fqdn?.message}
+            slotProps={{ inputRoot: { min: 1, max: 100 } }}
+          />
+          <div className="col-span-5 row-start-2">
+            <VerifyDomain
+              recordType="CNAME"
+              hostname={auth_fqdn}
+              value={`lb.${currentProject.region.domain}.`}
```
Comment (NesManrique) [inline dashboard/src/features/projects/custom-domains/settings/components/AuthDomain/AuthDomain.tsx:147]: This is still wrong, you are missing the region
```suggestion
              value={`lb.${currentProject.region.awsName}.${currentProject.region.domain}.`}
```

File: dashboard/src/features/projects/custom-domains/settings/components/HasuraDomain/HasuraDomain.tsx [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+            placeholder="auth.mydomain.dev"
+            error={Boolean(formState.errors.hasura_fqdn?.message)}
+            helperText={formState.errors.hasura_fqdn?.message}
+            slotProps={{ inputRoot: { min: 1, max: 100 } }}
+          />
+          <div className="col-span-5 row-start-2">
+            <VerifyDomain
+              recordType="CNAME"
+              hostname={hasura_fqdn}
+              value={`lb.${currentProject.region.domain}.`}
```
Comment (NesManrique) [inline dashboard/src/features/projects/custom-domains/settings/components/HasuraDomain/HasuraDomain.tsx:147]: This is still wrong, you are missing the region
```suggestion
              value={`lb.${currentProject.region.awsName}.${currentProject.region.domain}.`}
```

**Review by NesManrique — COMMENTED** [review]
Region still missing on the lb name

File: dashboard/src/features/projects/custom-domains/settings/components/AuthDomain/AuthDomain.tsx (hunk 1 above)
Comment (onehassan) [reply dashboard/src/features/projects/custom-domains/settings/components/AuthDomain/AuthDomain.tsx:118]: @nunopato Can you help out the copy?

File: dashboard/src/pages/[workspaceSlug]/[appSlug]/settings/custom-domains/index.tsx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+      className="grid max-w-5xl grid-flow-row gap-6 bg-transparent"
+      rootClassName="bg-transparent"
+    >
+      <Box className="flex flex-row items-center gap-4 overflow-hidden rounded-lg border-1 p-4">
+        <div className="flex flex-col space-y-2">
+          <Text className="text-lg font-semibold">Custom domains</Text>
+
+          <Text color="secondary">
+            Add a custom domain to Auth, Hasura, Run and even the database
+            service for only a $10 flat fee 🚀 <br /> Learn more about
```
Comment (nunopato) [inline dashboard/src/pages/[workspaceSlug]/[appSlug]/settings/custom-domains/index.tsx:45]:
```suggestion
            Add a custom domain to Auth, Hasura, PostgreSQL, and your Run 
            services for only a $10 flat fee 🚀 <br /> Learn more about
```

**Pull request merged**